Sources/PackageRegistry/RegistryManager.swift

```diff
@@ -135,9 +134,9 @@ public final class RegistryManager {
         var components = URLComponents(url: registry.url, resolvingAgainstBaseURL: true)
         components?.appendPathComponents("\(scope)", "\(name)", "\(version)", "Package.swift")
 
-        if let swiftLanguageVersion = swiftLanguageVersion {
+        if let toolsVersion = toolsVersion {
```

Comment:
So it's the tools version, not Swift version?

Reply:
yes, it should be tools version based afaik